saleor/graphql/channel/tests/queries/test_channel.py

```diff
@@ -19,18 +19,28 @@
             stockSettings{
                 allocationStrategy
             }
+            orderSettings {
+                automaticallyConfirmAllNewOrders
+                automaticallyFulfillNonShippableGiftCard
+            }
         }
     }
 """
 
 
-def test_query_channel_as_staff_user(staff_api_client, channel_USD):
+def test_query_channel_as_staff_user(
+    permission_manage_channels, staff_api_client, channel_USD
```

Comment:
IMO, we shouldn't extend this test but create the new one. Previously this test was used that staff user without permissions has access to basic channel data. Now, we're attaching `MANAGE_ORDERS`.

Reply:
Also, separate tests for staff/app that will confirm that without `MANAGE_ORDERS` we can't fetch `orderSettings` would be worth to have